### PR TITLE
Fixes all controls being set as links

### LINKED ISSUE #10619: A11y_AzureDataStudioStayHealthy_Welcome_AzureDataStudio-Insiders_AI4D: All controls are set as links.
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_AzureDataStudioStayHealthy;#A11ySev2;#A11yTCS;#A11yMAS;#Benchmark;#DesktopApp;#MAS4.1.2;#MAS1.3.1;#SQL Azure Data Studio;#FTP;#AI4D;#MAS4.1.2;#Win32;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.19.0-insider
Commit: e8dc0d1
Date: 2020-05-27T17:24:57.733Z
VS Code: 1.46.0
Electron: 7.2.4
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS References:MAS4.1.2, MAS1.3.1, MAS4.2.1
Tool: Accessibility insights for desktop

**Repro Steps:**
1 .Launch Azure Data Studio Insiders application.
2 . If Welcome screen is not shown on application launch, then navigate to Help --> Welcome.
3 . Launch accessibility insights for desktop and hover over the controls on the screen.

**Actual:**
Issue1:
On hovering over all the controls all of them are set as links, but on clicking each of them new dialog or tab have opened no redirection to a website.

Issue2:
On hovering over the new button the control is set as group and no name.

**Same issue in the following screen:**
1 .Resources-show more control 

**Expected:**
Issue1:
The control type for all controls should be set as a button and same should be announced by screen readers.
Since on activating each control, there is no redirection to a new url in the web browser.

Issue2:
The new control should be set as button with name as "New"

**User Impact:**
Screen reader users will get wrong control information of the control.

**Attachment link for Reference:**
A11y_AzureDataStudioStayHealthy_Welcome_AzureDataStudio-Insiders_AI4D
A11y_AzureDataStudioStayHealthy_Welcome_AzureDataStudio-Insiders_AI4D_Issue2

### PR DESCRIPTION
**Pull request by v-bbrady**
This PR fixes the accessibility bug that all the controls on the welcome page were set as links when they do not redirect you to another location. All the controls that perform commands within the ADS welcome page have now had added the role button attribute.

This PR fixes #10619

### CODE REVIEW

File: src/sql/platform/theme/common/colorRegistry.ts [hunk 1]
```diff
@@ -26,6 +26,7 @@ export const buttonSecondaryBorder = registerColor('button.secondaryBorder', { l
 export const buttonSecondaryBackground = registerColor('button.secondaryBackground', { light: null, dark: null, hc: null }, nls.localize('button.secondaryBackground', "The background color for the secondary button"));
 export const buttonSecondary = registerColor('button.secondaryForeground', { light: '#323130', dark: '#fff', hc: '#fff' }, nls.localize('button.secondaryForeground', "The font color for secondary button"));
 export const buttonSecondaryHoverColor = registerColor('button.secondaryHoverForeground', { light: '#0078D4', dark: '#3794ff', hc: '#3794ff' }, nls.localize('button.secondaryHoverForeground', "The hover color for secondary buttons"));
+export const buttonSecondaryHoverBorder = registerColor('button.buttonSecondaryHoverBorder', { light: '#0078D4', dark: '#3794ff', hc: '#3794ff' }, nls.localize('button.buttonSecondaryHoverBorder', "The hover border color for secondary buttons"));
```
Comment (alanrenmsft) [inline src/sql/platform/theme/common/colorRegistry.ts:29]: this doesn't seem to be related to the issue you are trying to fix

File: src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts [hunk 2]
```diff
@@ -51,7 +51,7 @@ export default () => `
 					</div>
 					<div class="row header-bottom-nav-tiles ads-grid">
 						<div class="col">
-							<a class="header-bottom-nav-tile-link" href="command:registeredServers.addConnection">
+							<a class="header-bottom-nav-tile-link" href="command:registeredServers.addConnection" role="button" name="Create a Connection">
```
Comment (alanrenmsft) [inline src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts:54]: is there any reason you are using name attribute? I think you should use aria-label here, also the text should be localized. this comment also applies to other changes in this PR.

Comment (v-bbrady) [reply src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts:54]: @alanrenmsft I was only doing this because of this comment left on the issue..

"The new control should be set as button with **name** as "New""

Comment (alanrenmsft) [reply src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts:54]: I am pretty sure they don't actually mean html name attribute. please use aria-label

File: src/sql/platform/theme/common/colorRegistry.ts (hunk 1 above)
Comment (v-bbrady) [reply src/sql/platform/theme/common/colorRegistry.ts]: removed @alanrenmsft

File: src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts (hunk 2 above)
Comment (v-bbrady) [reply src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts]: @alanrenmsft switched to aria-label

File: src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts [hunk 3]
```diff
@@ -13,14 +13,14 @@ export default () => `
...
 						<span role="tooltip" id="tooltip-text-wide" class="tool-tip-text" aria-hidden="true">
 							<h3 tabindex="0" class="preview-tooltip-header">${escape(localize('welcomePage.previewHeader', "This page is in preview"))}</h3>
 							<p tabindex="0" class="preview-tooltip-body">${escape(localize('welcomePage.previewBody', "Preview features introduce new functionalities that are on track to becoming a permanent part the product. They are stable, but need additional accessibility improvements. We welcome your early feedback while they are under development."))}</p>
 						</span>
 					</div>
 					<div class="tool-tip-container" id="tool-tip-container-narrow">
-						<a class="ads-welcome-page-link" aria-haspopup="true" class="preview-link" tabindex="0" id="preview-link-narrow" name="previewNarrow"><p>Preview</p><i class="icon-info themed-icon"></i></a>
+						<a class="ads-welcome-page-link" aria-haspopup="true" class="preview-link" tabindex="0" id="preview-link-narrow" aria-label="${escape(localize('welcomePage.previewNarrow', "Preview Narrow"))}"><p>Preview</p><i class="icon-info themed-icon"></i></a>
```
Comment (aaomidi) [inline src/sql/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts:23]: might be nice to have a function like `escapeLocalize`since this seems to be a pretty common use case

**Comment by alanrenmsft** [comment]
just like the other PR, please create a new pr to only include the fix for this issue mentioned in the PR description. @v-bbrady

**Comment by v-bbrady** [comment]
Created fresh branch to remove unneeded changes.
